### PR TITLE
Sprint14-apm-em-matwerk-pre1-issue98 and others

### PR DESCRIPTION
**Pull request by mkuehbach**
…r sprint14, pre1 for apm, em, and classes which can be useful for our colleagues in the NFDI-MatWerk consortium

### CODE REVIEW

File: contributed_definitions/NXsample.nxdl.xml
Comment (sanbrock) [inline contributed_definitions/NXsample.nxdl.xml:1]: NXsample, should be in base_classes/nyaml/

File: contributed_definitions/NXxpcs.nxdl.xml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
 <!--
 # NeXus - Neutron and X-ray Common Data Format
-#
-# Copyright (C) 2008-2022 NeXus International Advisory Committee (NIAC)
-#
+#
```
Comment (sanbrock) [inline contributed_definitions/NXxpcs.nxdl.xml:5]: It seems nxdl->nyaml was not generated with the latest version. Is it the case that we did not get this comment from the original nxdl file?

File: contributed_definitions/NXsample.nxdl.xml [hunk 2]
```diff
@@ -51,21 +72,21 @@
     <field name="chemical_formula" type="NX_CHAR">
         <doc>
              The chemical formula specified using CIF conventions.
-             Abbreviated version of CIF standard:
+             Abbreviated version of CIF standard':'
```
[the remark this answers is not in the document]
Comment (mkuehbach) [reply contributed_definitions/NXsample.nxdl.xml:75]: Indeed this is what nyaml2nxdl gave me when I compiled it in the morning with nomad-parser-nexus, (master, (https://github.com/nomad-coe/nomad-parser-nexus/commit/176451e5c38e3c307ce7384966d92d21cede1f3d), however while I worked on this, Sherjeel merged in the inheritance_check... branch resulting in that then nomad-parser-nexus master became 9b47a26. Lemme recompile that NXsample using the 9b47a26 version of nyaml2nxdl then I can be sure...

[the remark this answers is not in the document]
Comment (mkuehbach) [reply contributed_definitions/NXsample.nxdl.xml:75]: Indeed, even with 9b47a26 the formatting is CIF standard':' and not CIF standard:

**Comment by mkuehbach** [comment]
930aed741... was the nomad_parser_nexus commit used by @sanbrock to convert yaml to nxdl

File: contributed_definitions/NXsample.nxdl.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mkuehbach) [reply contributed_definitions/NXsample.nxdl.xml]: fixed

File: contributed_definitions/NXsample.nxdl.xml
Comment (mkuehbach) [reply contributed_definitions/NXsample.nxdl.xml:1]: fixed

File: contributed_definitions/NXxpcs.nxdl.xml (hunk 1 above)
Comment (mkuehbach) [reply contributed_definitions/NXxpcs.nxdl.xml]: after discussion with Sandor this week on his consolidation round plus my additional consolidation fixed

**Comment by mkuehbach** [comment]
@sanbrock all in now from my side

**Review by sanbrock — APPROVED** [review]

**Pull request merged**